### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by kristiyandobrev**
## Purpose

Updated `BuildableName` property for `ProdTestNet` scheme

### CODE REVIEW

**Review by abizjak — COMMENTED** [review]
The links for the stagenet should be updated. Instead of `eu.staging` it should be `stagenet` only.

**Comment by kristiyandobrev** [comment]
@abizjak 

Do you mean the proxy url ? 🤔

**Comment by abizjak** [comment]
> @abizjak
> 
> Do you mean the proxy url ? thinking

Yes.

**Comment by kristiyandobrev** [comment]
Updated.

File: ConcordiumWallet/Service/Network/ApiConstants.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ struct ApiConstants {
 #elseif MAINNET
     static let proxyUrl = URL(string: overriddenProxyUrl ?? "https://wallet-proxy.mainnet.concordium.software/v0")!
 #else // Staging
-    static let proxyUrl = URL(string: overriddenProxyUrl ?? "https://wallet-proxy.eu.staging.concordium.com/v0")!
+    static let proxyUrl = URL(string: overriddenProxyUrl ?? "https://wallet-proxy.staging.concordium.com/v0")!
```
Comment (abizjak) [inline ConcordiumWallet/Service/Network/ApiConstants.swift:21]:
```suggestion
    static let proxyUrl = URL(string: overriddenProxyUrl ?? "https://wallet-proxy.stagenet.concordium.com/v0")!
```

**Review by abizjak — COMMENTED** [review]
It's `stagenet` not `staging`.

**Review by abizjak — APPROVED** [review]

**Pull request merged**